### ISSUE TITLE
Option for Taskplay to disappear or grey out icons when there's no media input

### ISSUE THREAD

**Issue by Mancerrss** [evilpro/Taskplay#2]
It would be nice you'll implement some way for Taskplay to hide or disappear completely from the Tray to conserve Tray screen estate when there's no media input, or just an option for the icons to grey out (means they're disabled). I also suggest for a tooltip on the event saying "There's no media input" or something better UX English

**Comment by SpannerWZ** [comment]
@Mancerrss 

  this would be a useful feature, please consider it

**Comment by Gaurav-Original-ClassicShellTester** [comment]
Yes please. Taskbar screen estate is rare so it would be nice to free up space or automatically hide the controls when media stops. Also a stop button ◼️ would make it complete because some media players remember the last played position when simply paused. The stop button makes them not remember their position.

**Comment by dezhishen** [comment]
add option of default media input program
when there's no media input, start the program by the path.
